### PR TITLE
update readme for standardsettings 2.0

### LINKED ISSUE #13: update readme
i don't think it's a big deal to do it urgently since i can't think of anything that links to it right now

### PR DESCRIPTION
**Pull request by maskersss**
closes #13

### CODE REVIEW

**Comment by maskersss** [comment]
i removed the global standardoptions part as i assumed it wasn't implemented, let me know if i'm wrong

**Comment by contariaa** [comment]
it was implemented just not documented anywhere, you have to put a the redirect into .minecraft/config/mcsr/standardsettings.global

File: README.md [hunk 1]
```diff
@@ -2,87 +2,79 @@
...
-
-This file contains all the settings changed by StandardSettings, including most settings from Minecrafts options.txt file plus some additional settings. It does not include settings that aren't accessible from inside the game and some unnecessary ones (for example multiplayer related options) are also left out.
+*If you used StandardSettings before, you'd be used to the mod reading from `.../.minecraft/config/standardoptions.txt`. This is no longer the case!*
 
-To change your settings, you can either manually edit them in the file or set your settings ingame, then delete the file and restart Minecraft to generate a new one.
+To get to the config screen, go to `Options > Book and Quill > StandardSettings` in game. Here you can edit  all the settings changed by StandardSettings, including most settings from Minecraft's `options.txt` file, plus some additional settings. It does not include settings that aren't accessible from inside the game (i.e. difficulty – configure it in Atum's settings instead), and some unnecessary options (for example multiplayer related) are also left out.
```
Comment (contariaa) [inline README.md]: > some unnecessary options (for example multiplayer related) are also left out

Pretty sure it just covers every option now.

**Review by contariaa — CHANGES_REQUESTED** [review]
Could you explain F3 Pause on World Load and its delay on first world setting too?

**Comment by maskersss** [comment]
please fact check the "How can I use a global standardoptions file?" part

**Review by contariaa — APPROVED** [review]

**Comment by contariaa** [comment]
ty :)

**Pull request merged**